lib/crypto/tagger.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020-2022 CYBERCRYPT
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crypto
+
+import (
+	"bytes"
+	"encoding/gob"
+)
+
+const TagSize = 32
+const KeyLength = 32
```

Comment:
To be consistent and avoid confusion with other similar constants in the `crypto` package. 
```suggestion
const TagLength = 32
const TaggingKeyLength = 32
```